src/popup/bookmark.js

```diff
@@ -199,13 +200,14 @@ App.registerPanel(P_DETAILS, {
   _button: document.querySelector(".update-button"),
 
   async url() {
-    const {endpoint, email, apitoken} = await App.getSettings("endpoint", "email", "apitoken")
+    const {endpoint} = await App.getSettings("endpoint")
```

Comment:
is this a tuple with one element? isn't that just a variable?

Reply:
Nope :D `App.getSettings("endpoint")` returns a hash, and the `{endpoint}` syntax basically extracts that keys value into a top level variable https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment